hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/NetworkUtilizationManager.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.throttle;
+
+import com.hedera.node.app.state.HederaState;
+import com.hedera.node.app.workflows.TransactionInfo;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.time.Instant;
+
+public interface NetworkUtilizationManager {
```

Comment:
Can we add a similar comment as `NetworkUtilization`? It seems useful to understand what exactly this interface is for.